sap/5.5/modules/ROOT/pages/sap-connector-examples.adoc

```diff
@@ -690,6 +698,499 @@ output application/java
+. Click *Test Connection* to confirm that Mule runtime engine can connect with the SAP instance.
+. Enter `"BAPI_BANK_GETDETAIL"` for the *Function Name* and `payload` for the *Content*.
+. In the *Mule Palette* view, select *Core* and drag a *Logger* component next to *Synchronous Remote Function Call*.
+
+=== Configure the Second Flow
+
+The second flow creates a sales order from the data. To configure this flow:
+
+. From the *Mule Palette* view, select *HTTP* and drag the *Listener* operation to the canvas to start a new flow.
+. Configure the global element using the default values.
+. In the *Listener* properties tab, set the path to `/sync-mule-transaction`.
+. Drag a *Transform Message* component next to *Listener*.
++
+The content of this message creates a sales order from the data.
+. In the *Output* section of the *Transform Message* component, overlay the brackets with this text:
++
+----
+%dw 2.0
+output application/xml
+---
+{
+	BAPI_SALESORDER_CREATEFROMDAT2: {
+		"import": {
+			ORDER_HEADER_IN: {
+				DOC_TYPE: "TA",
+				SALES_ORG: "3000",
+				DISTR_CHAN: "10",
+				DIVISION: "00"
+			},
+			ORDER_HEADER_INX: {
+				DOC_TYPE: "X",
+				SALES_ORG: "X",
+				DISTR_CHAN: "X",
+				DIVISION: "X"
+			}
+		},
+		tables: {
+			ORDER_ITEMS_IN: {
+				row: {
+					ITM_NUMBER: "000010",
+					MATERIAL: "000000000050066128" as Number as String {format: "000000000000000000"},
+					PLANT: "3000",
+					TARGET_QTY: "1"
+				}
+			},
+			ORDER_ITEMS_INX: {
+				row: {
+					ITM_NUMBER: "000010",
+					MATERIAL: "X",
+					PLANT: "X",
+					TARGET_QTY: "X"
+				}
+			},
+			ORDER_PARTNERS: {
+				row: {
+					PARTN_ROLE: "WE",
+					PARTN_NUMB: "0000000448"
+				}
+			},
+			ORDER_SCHEDULES_IN: {
+				row: {
+					ITM_NUMBER: "000010",
+					SCHED_LINE: "0001",
+					REQ_QTY: "1"
+				}
+			},
+			ORDER_SCHEDULES_INX: {
+				row: {
+					ITM_NUMBER: "000010",
+					SCHED_LINE: "0001",
+					REQ_QTY: "X"
+				}
+			}
+		}
+	}
+}
+----
+
+. From the *Mule Palette* view, select *SAP* and drag the *Synchronous Remote Function Call* operation next to the *Transform Message* component.
+. Create a global element named `SAP_Config` and specify the connection information.
+. Click *Test Connection* to confirm that Mule runtime engine can connect with the SAP instance.
+. Enter `BAPI_SALESORDER_CREATEFROMDAT2` for the *Function Name* and `payload` for the *Content*.
+. From the *Mule Palette* view, select *Core* and drag *On Error Continue* under *Synchronous Remote Function Call*.
+. In the *Mule Palette* view, select *Core* and drag a *Logger* component into *On Error Continue*.
+. In the *Mule Palette* view, select *Core* and drag another *Logger* component next to *Synchronous Remote Function Call*.
+. Enter `payload.BAPI_SALESORDER_CREATEFROMDAT2.export.SALESDOCUMENT` for the *Message*.
+
+=== Configure the Third Flow
+
+The third flow creates a transaction ID and gets a list. To configure this flow:

```

Comment:
Async calls do not expect a response and require a different way of managing the transaction.  That's why we have specific operations for them. 
About the example, maybe doing a query may not be a useful scenario for an async. It would make more sense something that triggers a process or creates something on SAP.